nwb_web_gui/dashapps/pages/converter/converter_utils/forms.py

```diff
@@ -50,10 +50,7 @@ def __init__(self, required, fields, parent_name):
             label = dbc.Label(k)
             input_id = f'input_{parent}_{k}'
             explorer_id = input_id.replace('input', 'explorer')
-            if k in self.required_fields:
-                add_required = True
-            else:
-                add_required = False
+            add_required = k in self.required_fields
```

Comment:
You can assign the output of the logical test instead of using an `if` statement in these cases